src/components/CustomerQuota/types.ts

```diff
@@ -0,0 +1,3 @@
+export interface CartState {
+  [category: string]: boolean | null;
```

Comment:
Come to think of it, do we want a custom type for this?

"Selected" | "NotSelected" | "NoQuota"

Can make the code a lot easier to read also

Reply:
Yep I'll be changing the way the cart works so that it's clearer. As we're moving to allow numeric on top of boolean values, this will change too.